### PR TITLE
Mobile Banner Design Test

### PR DESCRIPTION
**Pull request by jlieb10**
## What does this change?
Introduce a mobile-only test to make sure that our new mobile banner design don't hurt our contributions

## Screenshots
Variant:
mobile_design_variant

Control:
mobile_design_control

## What is the value of this and can you measure success?
We will be measuring the results as Annualized Value (AV). If the variant doesn't hurt AV, that will become the control.

### Tested

- [X] Locally
- [ ] On CODE (optional)

### CODE REVIEW

File: static/src/javascripts/projects/common/modules/experiments/tests/contribs_global_mobile_banner_design.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    expiry: '2019-05-30',
+    author: 'Joshua Lieberman',
+    description: 'test new mobile design on standard acquisitions banner',
+    audience: 1,
+    audienceOffset: 0,
+    successMeasure: 'AV per impression',
+    audienceCriteria: 'All',
+    idealOutcome: 'variant design performs at least as well as control',
+    canRun: () =>
+        isBreakpoint({
```
Comment (joelochlann) [inline static/src/javascripts/projects/common/modules/experiments/tests/contribs_global_mobile_banner_design.js]: it's a real edge case, but how does it look if you load the page in a desktop browser shrunk down below the breakpoint requirement, but then expand out the browser to desktop width?

File: static/src/stylesheets/module/site-messages/_engagement-banner.scss [hunk 2]
```diff
@@ -194,3 +194,13 @@
         background: rgba($brightness-7,  .05);
     }
 }
+
+// For Mobile Design test:
+.engagement-banner__header {
+    @include fs-headline(3);
+    min-height: 50px;
+    font-weight: 700;
+    margin: 0 -10px $gs-baseline / 2;
```
Comment (joelochlann) [inline static/src/stylesheets/module/site-messages/_engagement-banner.scss]: `$gs-gutter / 2`

File: static/src/javascripts/projects/common/modules/commercial/templates/acquisitions-banner-mobile-design-test.js [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+    const applePayLogo = applePayApiAvailable ? applyPayMark.markup : '';
+    return `
+        <div class="engagement-banner__close">
+            <button class="button engagement-banner__close-button js-site-message-close js-engagement-banner-close-button" data-link-name="hide release message">
+                <span class="u-h">Close</span>
+                ${closeCentralIcon.markup}
+            </button>
+        </div>
+        <div class="engagement-banner__container">
+            <div class="engagement-banner__text engagement-banner__text--mobile">
```
Comment (joelochlann) [inline static/src/javascripts/projects/common/modules/commercial/templates/acquisitions-banner-mobile-design-test.js]: doesn't look like `engagement-banner__text--mobile` and `engagement-banner__header--mobile` have any styles associated with them?

File: static/src/javascripts/projects/common/modules/experiments/tests/contribs_global_mobile_banner_design.js (hunk 1 above)
Comment (jlieb10) [reply static/src/javascripts/projects/common/modules/experiments/tests/contribs_global_mobile_banner_design.js]: It's not bad:
mobile design edgecase

File: static/src/javascripts/projects/common/modules/experiments/tests/february-moment-banner.js [hunk 4]
```diff
@@ -1,152 +0,0 @@
-// @flow
-
-import { acquisitionsBannerFivTemplate } from 'common/modules/commercial/templates/acquisitions-banner-fiv';
```
Comment (joelochlann) [inline static/src/javascripts/projects/common/modules/experiments/tests/february-moment-banner.js]: personally I'd argue for removing `acquisitions-banner-fiv` and re-instating later if we need it, since it will affect the JS bundle  size and it's not clear that the design will be re-used as is

**Review by joelochlann — APPROVED** [review]
Looks good! Just wondering about the redundant classes?

File: static/src/javascripts/projects/common/modules/commercial/templates/acquisitions-banner-mobile-design-test.js (hunk 3 above)
Comment (jlieb10) [reply static/src/javascripts/projects/common/modules/commercial/templates/acquisitions-banner-mobile-design-test.js]: 👍 leftover from when i was trying to do both variants on one template

File: static/src/javascripts/projects/common/modules/experiments/tests/february-moment-banner.js (hunk 4 above)
Comment (jlieb10) [reply static/src/javascripts/projects/common/modules/experiments/tests/february-moment-banner.js]: 👍 will delete it

File: static/src/stylesheets/module/site-messages/_engagement-banner.scss (hunk 2 above)
Comment (jlieb10) [reply static/src/stylesheets/module/site-messages/_engagement-banner.scss]: 👍
